contracts/contracts/coordination/TACoChildApplication.sol

```diff
@@ -19,6 +19,7 @@ contract TACoChildApplication is ITACoRootToChild, ITACoChildApplication, Initia
         bool operatorConfirmed;
         uint96 authorized;
         // TODO: what about undelegations etc?
+        uint256 index; // index in stakingProviders array + 1
```

Comment:
I tried 2 slots and it gives almost no gas savings (< 100 gas in average tx). Because it mostly about how and when we access those slots and less slots not always better. But I don't mind to make those changes
✔️